### ISSUE TITLE
Unison keeps creating new archive files on every sync

### ISSUE THREAD

**Issue by llucps** [bcpierce00/unison#341]
Hi,

I've been using Unison since 2010 in all kind of environments such as Linux, OS X and FreeBSD.

Two years ago I decided to encrypt (gocryptfs) the folder that I sync between my laptop (OS X 10.15.4) and a remote server on FreeBSD with ZFS (I don't know the version, it's a third company). 

Anyway I've been happily using Unison 2.40.65 on my laptop and Unison 2.40.63 on the server, with no problems, only terminal no GUI.

But recently I ran into an issue which is running Unison from my laptop recognizes the replica on the server, that's all good, but then the Looking for changes and Waiting for changes from server gets stuck forever.

After troubleshooting I realized that for some reason when that happens a new archive file in .unison directory on the server and Library/Applications Support/Unison/ on OS X is created.. like if Unison wasn't able to see the previous archive files. 

The funny thing is that behavior is not consistent, sometimes and after fiddling a bit Unison runs just fine and detects the changes with no problem, although lately has been failing more than working.

I just wonder whether this could be a problem with using these "old" Unison versions 2.40.65? And updating to a newer version would help?

My profile file look like this.. nothing special I think:

# Unison preferences file
root = ssh://xxx@xxxx/doc/ (FreeBSD server)
root = /Users/myusername/.doc/ (laptop OS X)
ignore = Name *.tmp
ignore = Name Thumbs.db
ignore = Name ._*
ignore = Name .DS_Store
ignore = Path .TemporaryItems
ignore = Path .Trashes
ignore = Path .fseventsd
ignore = Path profiles
ignore = Path clients/xxxx/Virtualbox VM
fastcheck = true
log = false

Anyone has an idea where to look? 

Thanks

**Comment by llucps** [comment]
OK I think I found the problem... It seems the .fseventsd, .Trashes directories prevented Unison to run properly.. I really don't know why but as soon as I removed the ignored path in my profile file then Unison started to work again.

Although I don't want to sync those annoying files that OS X keeps adding, I followed the instructions from this link http://blog.hostilefork.com/trashes-fseventsd-and-spotlight-v100/

Basically I created:

.metadata_never_index (to avoid Spotlight indexing the directory)
.Trashes (file) (to stop OS X making .Trash directory)
.fseventsd/no_log (to stop OS X creating logging of filesystem events)

After doing this it seems is working again... but I'm really curious whether this is OS X problem using such an old 2.40.65 Unicson version? And if so, had it been fixed?

Thanks.
